release-notes/breaking-changes/elastic-apm.md

```diff
@@ -17,12 +17,19 @@ To learn how to upgrade, check out <uprade docs>.
 % **Action**<br> Steps for mitigating deprecation impact.
 % ::::
 
-% ## 9.0.0 [elastic-apm-900-breaking-changes]
-% **Release date:** March 25, 2025
+## 9.0.0 [elastic-apm-900-breaking-changes]
+**Release date:** March 25, 2025
 
 % ::::{dropdown} Title of breaking change 
 % Description of the breaking change.
 % For more information, check [PR #](PR link).
 % **Impact**<br> Impact of the breaking change.
 % **Action**<br> Steps for mitigating deprecation impact.
-% ::::
\ No newline at end of file
+% ::::
+
+::::{dropdown} Change server information endpoint / to only accept GET and HEAD requests
```

Comment:
It's a bit confusing in the dropdown title since we can't format as `/`.

<img width="802" alt="image" src="https://github.com/user-attachments/assets/7d8fd08c-a9a3-4e82-8c26-c72fd7c0300d" />

What do you think about this instead? 

```suggestion
::::{dropdown} Change server information endpoint "/" to only accept GET and HEAD requests
```